db/migrate/20210303074859_create_notes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateNotes < ActiveRecord::Migration
+  def change
+    create_table :notes do |t|
+      t.string :title, null: false
+      t.string :content, null: false
+      t.references :entity, polymorphic: true
```

Comment:
is it possible to have a note without an entity? should consider making it null: false?

```suggestion
      t.references :entity, polymorphic: true, null: false
```